cli/src/key_pair_storage.rs

```diff
@@ -147,15 +149,13 @@ fn get_or_create_path() -> Result<PathBuf, Error> {
     let path = path_buf.as_path();
     dir_ready(path.parent().unwrap().to_path_buf())?;
 
-    let old_path = build_path("accounts.json");
-    if !path.exists() && old_path.exists() {
-        println!("=> Migrating the key-pair storage to the latest version...");
-        std::fs::rename(old_path, path).map_err(WritingError::IO)?;
-        println!("✓ Done")
-    }
-
     if !path.exists() {
-        std::fs::write(path, b"{}").map_err(WritingError::IO)?;
+        let old_path = build_path("accounts.json");
+        if old_path.exists() {
+            std::fs::rename(old_path, path).map_err(WritingError::IO)?;
+        } else {
+            std::fs::write(path, b"{}").map_err(WritingError::IO)?;
```

Comment:
With `b"{}"` we hard-code the serialization of an empty, unversioned key pair storage (i.e. `FileSchema::Unversioned(HashMap::new()))`. I think it would be less brittle to write the serialized version of the latest empty key pair storage.
```suggestion
            let empty = FileSchema::Versioned(VersionedSchema::V1 { key_pairs: HashMap::new() });
            std::fs::write(path, b"{}").map_err(WritingError::IO)?;
```
Of course that one line should be extracted.

Reply:
Yeah... I wanted to simply call `update(HashMap::new())?`, but `update` calls `get_or_create_path`, wich would then call `update`, resulting in a stack overflow.

Going with your suggestion, we would also need to json serialize the empty V1:

```diff
-   std::fs::write(path, b"{}").map_err(WritingError::IO)?;
+   let empty = FileSchema::latest_empty();
+   let empty_json =
+      serde_json::to_string_pretty(&empty).map_err(WritingError::Serialization)?;
+   std::fs::write(path, empty_json).map_err(WritingError::IO)?;
```
I didn't find the added noise and complexity worth it. Do you disagree? I'd prefer to leave it as is and, when we tackle the follow-up issues reported in the description above, just call `update(HashMap::new())?`.